dataprofiler/profilers/categorical_column_profile.py

```diff
@@ -76,6 +81,9 @@ def profile(self):
             profile["statistics"].update(
                 dict(categories=self.categories)
             )
+            profile["statistics"]['categorical_count'] = dict(
```

Comment:
Should the name "categorical_count" be elaborated on to specify that not all categories will be present in this statistic, only the top 5. e.g. "top_categorical_counts" or "top_categories" or something of the sort?

Reply:
interesting thought, maybe refactor after this PR?